TSG/Update/manual-retry-after-caupostcheck.md

```diff
@@ -0,0 +1,74 @@
+
+   Once you get the LastUpdate xml file, if the status of step '20.20.8.20.20.4' is not 'Error', you might collect the wrong xml file. You could use below command listing all action instance and find the one which 'actionplanname' is 'MAS Update' and the time is matched with last failure. The status of the action plan should be 'Failed':
+
+   `Get-actionplaninstances -ececlient $ececlient | ft status, startdatetime, enddatetime, instanceid, actionplanname`
+
+   Then use that 'actionplaninstancid' to save the correct LastUpdate xml file again by using this command:
+
+   `$(get-actionplaninstance -actionplaninstanceid <instanceid> -ececlient $ececlient).ProgressAsXml | Out-File -FilePath c:\temp\LastUpdate.xml`
+
+3. Open the LastUpdate.xml file in c:\temp folder and find the step '20.20.8.20.20.4'. Replace the whole content of step '20.20.8.20.20.4' by this content:
+
+   ```
+   <!-- Update includes host reboot -->
+   <Step FullStepIndex="20.20.8.20.20.4" Index="4" Name="Update Host OS" StartTimeUtc="<keep the original start time>" Status="Error" EndTimeUtc="<Keep the origional end time>">
+     <Task RolePath ="CAU" ActionType ="ClusterAwareUpdateForOSUpdate" />
+   </Step>
+   ```
+
+4. Trigger the custom action plan by using command:
+
+   `$instanceguid = Invoke-ActionPlanInstance -ActionPlanPath c:\temp\LastUpdate.xml -ececlient $ececlient`
+
+5. Track the progress of the custom action plan by using command:
+
+   `get-actionplaninstance -actionplaninstanceid $instanceguid -ececlient $ececlient`
+
+## What to do if the final status is 'Failed'
+Once the action plan finished. If the final status is 'Failed', you could use below command to collect the latest update xml file again:
+
+`$LastUpdate = Get-ActionPlanInstances -ececlient $ececlient | Where-Object ActionPlanName - 
+ match "MAS Update" | Sort-Object -Property EndDateTime -Descending | Select-Object -First 1`
+
+`$LastUpdate.ProgressAsXml | Out-File c:\temp\LatestUpdate.xml`
+
+Open the LatestUpdate.xml, if the error is still within step '20.20.8.20.20.4'. You could use below command to collect CAU related log and attached it to the ICM (note that this cmdlet saves a zip file to current directory as CAUDebugTrace.zip; to save at custom location use -FilePath parameter):
+
+`$clusterName = Get-Cluster`
+`Save-CauDebugTrace -ClusterName $clusterName -FeatureUpdateLogs All -Force`
+
+If the failure is not in step '20.20.8.20.20.4', you might create a new ICM and find the correct owner of that interface.
+
+# Escalate to SME or Engineering
+If you are stalled on troubleshooting or customer has requested an official RCA statement, you will need to ensure that you have the appropriate information gathered before escalating to a SME, TA, EEE or submitting an ICM.
+
+Document all the actions taken including date and time.
+
+Refer to [Processes](/Processes.md) for information on how to locate the appropriate SMEs or Engineering team.
```

Comment:
We don't have this "Processes" file in our repo so we should probably remove the reference to it.